Combat/Include/MonsterCard.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "Card.hpp"
+#include <iostream>
+
+class MonsterCard : public Card
+{
+    protected:
+        int hp;
+        int strength;
+    public:
+        MonsterCard(int = 20,int = 10);
+        std::string useCard() override;
+        int takeDamage(int damagePoints);
+};

```

Comment:
Nadal brak enterów na końcu, to o co na slacku pytałem . Podrzucam link https://stackoverflow.com/questions/30403150/atom-text-editor-remove-trailing-whitespace-on-save

Reply:
> Nadal brak enterów na końcu, to o co na slacku pytałem . Podrzucam link https://stackoverflow.com/questions/30403150/atom-text-editor-remove-trailing-whitespace-on-save

@JakubDryka Wydaje mi się, że jest dobrze. To ma być pojedynczy newline, który często nie jest wyświetlany